hack/cozyreport.sh

```
@@ -157,8 +231,28 @@ if kubectl get deploy -n cozy-linstor linstor-controller >/dev/null 2>&1; then
+# -- sandbox-host module
+
+echo "Collecting sandbox host context..."
+DIR=$REPORT_DIR/sandbox-host
+mkdir -p $DIR
+df -h > $DIR/df.txt 2>&1
+free -m > $DIR/free.txt 2>&1
+ps auxww > $DIR/ps.txt 2>&1
+dmesg | tail -200 > $DIR/dmesg.txt 2>&1 || true
+if [ -f /workspace/talosconfig ]; then
+  for node in 192.168.123.11 192.168.123.12 192.168.123.13; do
+    talosctl --talosconfig /workspace/talosconfig -n $node dmesg --tail=200 > $DIR/talos-$node-dmesg.txt 2>&1 || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)